src/pages/Filing/FilingApp/FilingSubmit.tsx

```diff
@@ -116,8 +121,30 @@ export function FilingSubmit(): JSX.Element {
             <TextIntroduction
               heading='Sign and submit'
               subheading='Before you sign and submit, carefully review all the information provided in each of the following sections. For each section, check the box if the information is complete and accurate, or follow the instructions to make changes.'
-              description='An authorized representative of your financial institution with knowledge of the data must certify the accuracy and completeness of the data reported pursuant to § 1002.109(a)(1)(ii).'
+              description={
+                <p>
+                  An authorized representative of your financial institution
+                  with knowledge of the data must certify the accuracy and
+                  completeness of the data reported pursuant to{' '}
+                  <Links.FIG section='§ 1002.109(a)(1)(ii)' />.
+                </p>
+              }
             />
+            <Alert
+              status='warning'
```

Comment:
Can we get an `aria-live=polite` [on this](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions) alert box? I don't want to yell at them about it, but it's a nice thing to do before they try to skip into a disabled form.